common/src/storage/complex/messenger.rs

```diff
@@ -165,7 +165,10 @@ pub trait Messenger {
             Key = Self::WaitlistFirstKey,
         >;
 
-    type DispatchStash: MapStorage<Key = Self::DispatchStashKey, Value = Self::QueuedDispatch>;
+    type DispatchStash: MapStorage<
+        Key = Self::DispatchStashKey,
+        Value = (Self::QueuedDispatch, Self::BlockNumber),
```

Comment:
We had a conversation on using `Interval` here, isn't it?
```suggestion
        Value = (Self::QueuedDispatch, Interval<Self::BlockNumber>),
```

Reply:
done